README.md

```diff
@@ -139,7 +140,7 @@ Examples:
 ```
 
 # Programmatic Usage
-See `node_modules/electron-builder/out/electron-builder.d.ts`. [Typings](https://github.com/Microsoft/TypeScript/wiki/Typings-for-npm-packages) is supported.
+See `node_modules/electron-builder/out/electron-builder.d.ts`. [Typings](https://github.com/Microsoft/TypeScript/wiki/Typings-for-npm-packages) are supported.
```

Comment:
`is` is correct :) singular — https://github.com/typings/typings — `Typings is the simple way to manage and install TypeScript definitions.`.